app/views/users/index.html.erb

```diff
@@ -1 +1,15 @@
-Here is a list of users
\ No newline at end of file
+<% if @users.empty? %>
+   <h1>No users</h1>
+<% else %>
+    <ul class='users_list'>
+      <% @users.each do |user| %>
+        <li class='user_card'> 
+          <img src=<% user.photo %> />
+          <div>
+            <h3><%= user.name %></h3>
+            <p>Number of Posts: <%= user.posts_counter %></p>
+          </div>
+        </li>
+      <% end %>  
+    </ul>
```

Comment:
- [ ] as mentioned by the previous reviewer, Kindly ensure that Clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png). currently clicking on users doesn't redirect me to the user view page